app/src/app/admin/articles/article/article.ts

```diff
@@ -78,12 +106,13 @@ namespace app.admin.articles.article {
 
     export class ArticleController {
 
-        static $inject = ['article', '$stateParams', 'articleService', 'notificationService'];
+        static $inject = ['article', '$stateParams', 'articleService', 'notificationService', 'articleMetaTags'];
         constructor(
             public article:common.models.Article,
             public $stateParams:IArticleStateParams,
             private articleService:common.services.article.ArticleService,
-            private notificationService:common.services.notification.NotificationService) {
+            private notificationService:common.services.notification.NotificationService
+        ) {
         }
```

Comment:
This controller isn't using articleMetaTags, so it shouldn't be injected - it is better to inject it into the controller that actually needs it.